addons/website_blog/static/src/js/options.js

```diff
@@ -39,6 +39,28 @@ options.registry.many2one.include({
     }
 });
 
+options.registry.ContainerWidth.include({
+    /**
+     * @override
+     */
+    selectClass: async function (previewMode, widgetValue, params) {
+        await this._super(...arguments);
+        // Apply change to breadcrumb or tags if applicable.
+        const widthClasses = ['container', 'container-fluid', 'o_container_small'];
+        if (widthClasses.includes(widgetValue)) {
+            const section = this.$target[0].closest('section');
+            const sections = section.closest('#o_wblog_post_content').querySelectorAll('section');
+            const sectionIndex = [...sections].indexOf(section);
+            const impacted = sectionIndex === 0 ? ['#o_wblog_post_content .breadcrumb'] :
+                sectionIndex === sections.length - 1 ? '.o_wblog_post_tags' : undefined;
+            document.body.querySelectorAll(impacted).forEach(container => {
+                container.classList.remove(...widthClasses.filter(c => c !== widgetValue));
+                container.classList.add(widgetValue);
+            });
+        }
+    },
+});
```

Comment:
Did not read in details but that does not seem to bad for the faeture to be done, so we can keep in this direction :+1: